### PR TITLE
Reset form data 

### PR DESCRIPTION
**Pull request by OtienoSamwel**
Function to reset form data to initial values

### CODE REVIEW

File: form-builder/src/main/java/com/dsc/form_builder/ChoiceState.kt [hunk 1]
```diff
@@ -20,7 +20,7 @@ package com.dsc.form_builder
 class ChoiceState(
     name: String,
     initial: String = "",
-    validators: List<Validators>,
+    validators: List<Validators> = listOf(),
```
Comment (LinusMuema) [inline form-builder/src/main/java/com/dsc/form_builder/ChoiceState.kt:23]: Just curious, why did you add a default list of validators?

**Review by LinusMuema — CHANGES_REQUESTED** [review]
Hi @OtienoSamwel 

Thanks for the changes. I only have one suggestion

File: form-builder/src/main/java/com/dsc/form_builder/ChoiceState.kt (hunk 1 above)
Comment (OtienoSamwel) [reply form-builder/src/main/java/com/dsc/form_builder/ChoiceState.kt:23]: For uniformity mainly. The rest of the fields have a default list of validators. Unless there is a specific reason to enforce validations on this class?

File: form-builder/src/test/java/com/dsc/form_builder/TextFieldStateTest.kt [hunk 2]
```diff
@@ -125,4 +125,32 @@ internal class TextFieldStateTest {
             assert(actual == expected)
         }
     }
+
+    @Nested
+    inner class DescribingFormState {
```
[the remark this answers is not in the document]
Comment (OtienoSamwel) [reply form-builder/src/test/java/com/dsc/form_builder/TextFieldStateTest.kt:130]: There are really good ideas. Let me do that.

File: form-builder/src/main/java/com/dsc/form_builder/ChoiceState.kt (hunk 1 above)
Comment (LinusMuema) [reply form-builder/src/main/java/com/dsc/form_builder/ChoiceState.kt:23]: Oh...it's fine.
Good catch
